includes/rest-api/class-sensei-rest-api-extensions-controller.php

```diff
@@ -350,8 +350,8 @@ private function check_for_upgrade_error( array $plugins, $result, WP_Ajax_Upgra
 	private function create_extensions_response( array $plugins, string $extensions_key, bool $full_response = false ): WP_REST_Response {
 		$mapped_plugins = array_map(
 			function ( $plugin ) {
-				$plugin->price      = html_entity_decode( $plugin->price );
-				$plugin->image      = $plugin->image_large;
+				$plugin->price      = isset( $plugin->price ) ? html_entity_decode( $plugin->price ) : '';
```

Comment:
WooCommerce doesn't have this information that comes from senseilms.com API.

Reply:
WooCommerce isn't free? Maybe we can assume that not having price is same as free.

Reply:
Yep! I think so! So for WC, it will have an empty string as the price string.

Basically, I added that check otherwise, it would fail when trying to run `html_entity_decode` against an unset variable.